hadoop-hdds/client/src/main/java/org/apache/hadoop/hdds/scm/storage/BlockInputStream.java

```diff
@@ -302,20 +310,24 @@ protected synchronized int readWithStrategy(ByteReaderStrategy strategy)
       try {
         numBytesRead = strategy.readFromBlock(current, numBytesToRead);
         retries = 0; // reset retries after successful read
+      } catch (SCMSecurityException ex) {
```

Comment:
Whats the context for this change?

Reply:
It was there. I just move that piece up to separate it from other exceptions that we retry on.